watch.go

```diff
@@ -180,7 +204,8 @@ func (fb *Firebase) Watch(notifications chan Event) error {
 
 				// TODO: handle
 			case "rules_debug":
-				log.Printf("Rules-Debug: %s\n", txt)
+				txt, _ := ioutil.ReadAll(resp.Body)
+				log.Printf("Rules-Debug: %s\n", string(txt))
```

Comment:
I'll go ahead and run a test for this. My fear with reading the entire response body is that it will read until it get's an `EOF` which may cause other messages to be missed.